### PR TITLE
Contact us page

### LINKED ISSUE #487: Funcionalidade de Fale conosco
## Feature Description

Deve ser implementado uma feature de Fale Conosco para permitir qualquer usuário entrar em contato com a administração da plataforma.

- Precisa encontrar o melhor local dentro do layout para inserir o link ou ícone do Fale Conosco
- Abrir modal com os campos:
   - Nome
   - Email
   - Telefone (opcional)
   - Assunto (lista suspensa com as opções: sugestão de melhoria, reportar erro, reportar abuso, crítica)
   - Mensagem
- Botão de enviar 

---------------

## Acceptance criteria

* 
- Qualquer usuário pode enviar uma mensagem ao Fale Conosco
- Não precisa ficar armazenada no banco essas mensagens. Toda a troca de conversa será por meio do email.
- Usuário e admin (plataformasabia@ufersa.edu.br) recebem email de confirmação
- Ao enviar apresentar mensagem de confirmação de envio

## Implementation Brief

*

### PR DESCRIPTION
**Pull request by luizeboli**
## Summary

Resolves #487

image

## Relevant technical choices

* As you can see I've added some variants to our form components, as `InputField`, `TextField` and `SelectField`. This will help us on developing new screens with updated style guide. 
* **The API isn't done yet, we'll integrate later**

## QA Steps

1. Access home page
2. Click `Contato` link on page header

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

File: packages/web/components/Form/styles.js [hunk 1]
```diff
@@ -17,24 +17,36 @@ export const Row = styled.div`
...
-		color: ${colors.lightGray};
 		opacity: ${disabled ? 0.5 : 1};
 
 		&::placeholder {
 			color: ${colors.lightGray2};
 			font-weight: 300;
 			font-style: italic;
 		}
+
+		${!!variant && inputModifiers[variant]({ colors, metrics })}
```
Comment (lcnogueira) [inline packages/web/components/Form/styles.js:49]: Willian's ideas, right @luizeboli ? 😄

File: packages/web/components/Modal/ContactUsSuccessModal/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	return (
+		<Container>
+			<img
+				src="charity-rafiki.svg"
+				alt="Ilustração de uma garota dentro de um smartphone recebendo uma doação de um garoto"
+			/>
+
+			<div>
+				<h3>Recebemos sua mensagem!</h3>
+				<p>Obrigado pelo seu contato, assim que possível, retornaremos.</p>
```
Comment (lcnogueira) [inline packages/web/components/Modal/ContactUsSuccessModal/index.js:14]:
```suggestion
				<p>Obrigado pelo seu contato! Retornaremos assim que possível.</p>
```

File: packages/web/pages/contact-us.js [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+	> img {
+		margin-left: 8.8rem;
+	}
+`;
+
+const StyledTitle = styled(Title)`
+	margin-bottom: 3.2rem;
+`;
+
+const CancelButton = styled(Button)`
```
Comment (lcnogueira) [inline packages/web/pages/contact-us.js:168]: Aren't those common buttons used in some screens? If so, we should just turn it into a component in order to make it reusable...

File: packages/web/pages/contact-us.js [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+						name="user.email"
+						type="email"
+						label="Seu melhor e-mail"
+						validation={{ required: true }}
+						wrapperCss={inputWrapperCss}
+						variant="gray"
+					/>
+
+					<InputField
+						name="user.phone"
```
Comment (lcnogueira) [inline packages/web/pages/contact-us.js:82]: I think we might consider using the `MaskedInputField` here to turn this into a masked input. There is a good example here: https://github.com/ufersa/plataforma-sabia/blob/2a232df29fa3fd4b59162edaf40896a2cba5b56a/packages/web/components/TechnologyForm/Responsible/index.js#L123. Take a look to see if it's not too much trouble...if not, switch it for the MaskedInputField 👍

**Review by lcnogueira — CHANGES_REQUESTED** [review]
Great job, so far. Requested a few changes in order to make it even better 👍

File: packages/web/pages/contact-us.js (hunk 3 above)
Comment (luizeboli) [reply packages/web/pages/contact-us.js:168]: @lcnogueira Yes, these buttons are used in a bunch of screens, but we decided to refactor it after releasing, with interface refactor too. That's why I'm reusing it from other screens.

Comment (luizeboli) [reply packages/web/pages/contact-us.js:168]: We talked about that when me and @mateus4k were developing a screen and needed to use the same button styles, but it didn't exist yet. We agreed to continue and refactor later 😄

File: packages/web/components/Form/styles.js (hunk 1 above)
Comment (luizeboli) [reply packages/web/components/Form/styles.js:49]: For sure 😛

File: packages/web/components/Modal/ContactUsSuccessModal/index.js (hunk 2 above)
Comment (luizeboli) [reply packages/web/components/Modal/ContactUsSuccessModal/index.js:14]: Do you agree @JoaoPauloSMoura ?

File: packages/web/pages/contact-us.js (hunk 4 above)
Comment (luizeboli) [reply packages/web/pages/contact-us.js:82]: Easy peasy, thanks for suggesting that 😛 

Oct-29-2020 14-14-28

**Comment by luizeboli** [comment]
Hey @lcnogueira thanks for your points, I'm ready for the second round 😄

File: packages/web/pages/contact-us.js (hunk 3 above)
Comment (lcnogueira) [reply packages/web/pages/contact-us.js:173]: Ok, got it. Makes sense 👍

**Review by lcnogueira — APPROVED** [review]
LGTM 👊 👊

**Review by mateus4k — APPROVED** [review]
LGTM 👍

**Pull request merged**